src/richie/plugins/nesteditem/templates/richie/nesteditem/nesteditem.html

```diff
@@ -0,0 +1,91 @@
+{% load i18n cms_tags %}{% spaceless %}
+
+{% with variant=variant|default:"default" %}
+
+{% comment %}Accordion variant to make first level container, always turn to
+an "accordion_item" variant for second level.{% endcomment %}
+{% if variant == "accordion" %}
+    <div class="nesteditem nesteditem--{{ variant }}">
+        {% if instance.content %}
+            <h{{ header_level|default:2 }} class="nesteditem__head">
+                {{ instance.content|linebreaksbr }}
+            </h{{ header_level|default:2 }}>
+        {% endif %}
+        {% if instance.child_plugin_instances %}
+        <div class="nesteditem__items">
+            {% for plugin in instance.child_plugin_instances %}
+                {% with variant="accordion_item" %}
+                    <div class="nesteditem__item" data-accordion-item>
+                    {% render_plugin plugin %}
+                    </div>
+                {% endwith %}
+            {% endfor %}
+        </div>
+        {% endif %}
+    </div>
+{% comment %}Accordion item, used automatically from "accordion" variant, this
+is for folding item. It is not intended to be used "manually". Items will be
+turned as a simple list using "list" variant.{% endcomment %}
+{% elif variant == "accordion_item" %}
```

Comment:
I didn't see this variant :man_shrugging: in choices

Reply:
This is a private variant to be used only from the main "accordion" variant. As you may saw from the source code, accordion require a structure with a container to surround items. I may have included this in the main "accordion" variant with some "if" and an new private context variable but i didn't know if it have been ok for you.

Reply:
then there is no reason to share the same template... rename this template (and plugin?) to nested_group.html or nested_structure.html and add a separate template nested_item.html?